lib/src/getZodiosEndpointDefinitionList.ts

```diff
@@ -130,14 +131,19 @@ export const getZodiosEndpointDefinitionList = (doc: OpenAPIObject, options?: Te
     const ignoredGenericError = [] as string[];
 
     for (const path in doc.paths) {
-        const pathItem = doc.paths[path] as PathItemObject;
+        const pathItemObj = doc.paths[path] as PathItemObject;
+        const pathItem = omit(pathItemObj, ["parameters"]);
+        const parametersMap = getParametersMap(pathItemObj.parameters ?? []);
 
         for (const method in pathItem) {
-            const operation = pathItem[method as keyof PathItemObject] as OperationObject;
+            const operation = pathItem[method as Exclude<keyof PathItemObject, "parameters">] as OperationObject;
```

Comment:
I'm not quite sure if a blacklist of the parameters key is the best solution here.

The OpenAPI spec allows other keys (like `summary`, `description` and even `servers`). Also extensions can be used to add other fields in this object.
I would suggest to use a whitelist of the http operations `get`,`put`,`post`,`delete`,`options`,`head`,`patch` and `trace` as defined in the [spec](https://swagger.io/specification/#path-item-object).